src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -288,6 +289,115 @@ public void SetThreadPrincipal(IPrincipal principal)
             }
         }
 
+        public ObjectHandle CreateInstance(string assemblyName, string typeName)
+        { 
+            if (assemblyName == null)
+                throw new ArgumentNullException(nameof(assemblyName));
```

Comment:
can you please add the braces here to have uniformity with the other exceptions being thrown in this file

Reply:
done